main.go

```diff
@@ -783,6 +822,43 @@ var DevCopyCommand = cli.Command{
 	},
 }
 
+func genLockDeps(pkg *Package, deps map[string]gx.LockDep, done map[string]bool, ignoreConflict bool) error {
```

Comment:
Could we use the [`DependencyQueue`](https://github.com/whyrusleeping/gx/blob/855ebcf2e2e25d7d55f28f5c35044e8e75264c17/gxutil/pm.go#L695) here? (I'm not sure, since it belongs to the `gx` repo).

Ideally I would like to have a function that just returns all the (unique) dependencies of a package. From what I'm seeing in the gx/gx-go code bases we're doing a lot of "get me all your direct and transitive dependencies" (converting `Dependency` to `Package`) types of operation.